chat/prekey/key_test.go

```diff
@@ -72,3 +73,10 @@ func TestPreKey_ToProtobufAndBack(t *testing.T) {
 	require.Equal(t, hex.EncodeToString(pp.Key), hex.EncodeToString(k.PublicKey[:]))
 
 }
+
+func TestPreKey_OlderThen(t *testing.T) {
```

Comment:
OlderThen should be renamed to OlderThan to be grammatically correct.

Reply:
Done